config/commit-template

```diff
@@ -0,0 +1,5 @@
+The title
```

Comment:
100% not adding a 'Description:' as its already a given

I'm fine with doing Reviewers: but today we dont have a lot of value and all reviewers are signaled with @mention